### PR TITLE
New Gilt HQ

### PR DESCRIPTION
**Pull request by johncoghlan**
Wrote a blog post about the new office. Take a look and let me know what you think!

🗽 🍾 🍉 😀

### CODE REVIEW

File: _posts/2016-08-29-new-gilt-hq.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+Everyone is also really stoked about the views. The floorplan was designed to maximize the views and natural light by keeping the workstations and common areas on the perimeter while offices and conference rooms are in the center of the floor. Our location on the water provides great (I mean _really_ great) views of the Statue of Liberty, New York Harbor, the Hudson River, Lower Manhattan and New Jersey. 
+
+<p align="center">
+  <img src="http://i.imgur.com/6WYb4lt.jpg")
+</p>
+
+The floor has dozens of conference rooms, a pantry, two large collaboration spaces and room for about 320 employees. Our ping pong and foosball tables will be arriving shortly. 
+
+The design of the space is clean and mininmalist with lots of white and grey touches. Everyone is loving their new super comfy Herman Miller Aeron chairs and the cool little sofas in the collaboration spaces. We have some nice portraits on the walls already and our creative teams are working on a mural for one of the bigger walls to give the floor some more character.
```
Comment (zcohn) [inline _posts/2016-08-29-new-gilt-hq.md]: mininmalist->minimalist

File: _posts/2016-08-29-new-gilt-hq.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## The neighborhood 
+
+Brookfield Place is an awesome place to work. We're right on the water and there is a marina with a sailing school and a handful of yachts right outside our doors. Our building is surrounded by grassy parks with volleyball courts, basketball courts and a skatepark in walking distance. The waterfont is car-free which gives the area a quiet and peaceful vibe.
```
Comment (zcohn) [inline _posts/2016-08-29-new-gilt-hq.md]: waterfont->waterfront

File: _posts/2016-08-29-new-gilt-hq.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+When it comes to food, the variety is amazing. In our building alone we have the huge French marketplace Le District and Hudson Eats, a food hall with a bunch of NYC hot spots like Chop't, Mighty Quinn's, Num Pang, Umami Burger, Dos Toros and Blue Ribbon to name a few. Around the neighborhood, we have Shack Shack, Parm, a bunch of great coffee shops, multiple Starbucks (of course) and Eataly, which just opened across the street. 
+
+Other area amenities include an Equinox gym, tons of shopping, waterfront bars, Citibike stations, bike parking in our building and a ton of transit options. 
+
+## The welcome
+
+Of course, a new office needs to get broken in and People Operations was on it. We had breakfasts, lunch, a champagne toast and a watermelon social to help keep the excitement level high and give everyone a chance to meet their new neighbors since this was the first time our Gilt and HBC teams would be fully intergrated on the same floor. The events were a hit and everyone seems be settling in nicely. Mission accomplished.
```
Comment (zcohn) [inline _posts/2016-08-29-new-gilt-hq.md]: intergrated->integrated

File: _posts/2016-08-29-new-gilt-hq.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+Woohoo! Gilt and our colleagues at HBC Digital just moved into brand new offices! Our new digs are in Brookfield Place, a complex of office buildings across West Street from the Freedom Tower in Lower Manhattan. There's a lot to love about the new space. 
+
+<p align="center">
+<img src="http://i.imgur.com/hos6HZw.jpg")
+</p>
+
+## The office
+
+It is great to have (almost) all of HBC and Gilt in one location (we have teams in Dublin, St. Louis and Jackson and some remote engineers). We're confident working side-by-side will allow us to better integrate our teams and better leverage our strengths.
```
Comment (zcohn) [inline _posts/2016-08-29-new-gilt-hq.md]: It's at least great to have the NYC-based folks in one place :-). Gilt tech is still 50% Dublin, though!

**Comment by zcohn** [comment]
👍  -- and I'm also looking forward to having some meet-ups in the new space!

**Comment by zcohn** [comment]
👊

**Comment by johncoghlan** [comment]
Want to merge today - any other feedback before we merge?

**Pull request merged**